ui/tests/acceptance/console-test.js

```diff
@@ -45,4 +45,39 @@ module('Acceptance | console', function(hooks) {
       assert.equal(consoleEle.offsetTop, 0, 'fullscreen is aligned to the top of window');
     }, 300);
   });
+
+  test('array output is correctly formatted', async function(assert) {
+    await consoleComponent.toggle();
+    await consoleComponent.runCommands('read -field=policies /auth/token/lookup-self');
+
+    // have to wrap in a later so that we can wait for the CSS transition to finish
+    await later(() => {
+      let consoleOut = document.querySelector('.console-ui-output>pre').innerText;
+
+      assert.notOk(consoleOut.includes('function(){'));
+      assert.equal(consoleOut, '["root"]', 'Unexpected content:' + consoleOut);
```

Comment:
The optional message argument for qunit assertions are just meant as descriptions for the assertions, so it's odd to have the test pass with `unexpected content: ` and then print out the expected content. Probably easier to leave that argument out for now. I _think_ qunit should diff the first two args in this case and then display a diff.

Reply:
Ah, that's my unfamiliarity with qunit showing! Adding that to the PR now...

Reply:
Awesome, thanks for making that change!